public/js/helpers/humanize_helper.js

```diff
@@ -160,8 +160,10 @@ var humanize = {
   },
   date: function (stamp, withTimezone) {
     var d = new Date(stamp)
-    return `${String(d.getUTCFullYear())}-${String(d.getUTCMonth() + 1).padStart(2, '0')}-${String(d.getUTCDate()).padStart(2, '0')} ` +
-          `${String(d.getUTCHours()).padStart(2, '0')}:${String(d.getUTCMinutes()).padStart(2, '0')}:${String(d.getUTCSeconds()).padStart(2, '0')} (UTC)`
+    var dateStr = `${String(d.getUTCFullYear())}-${String(d.getUTCMonth() + 1).padStart(2, '0')}-${String(d.getUTCDate()).padStart(2, '0')} ` +
+          `${String(d.getUTCHours()).padStart(2, '0')}:${String(d.getUTCMinutes()).padStart(2, '0')}:${String(d.getUTCSeconds()).padStart(2, '0')}`
+    if (withTimezone) dateStr += ' (UTC)'
+    return dateStr
```

Comment:
This seems reasonable to me, although since previously `withTimezone` was being dropped, I wonder if there was a latent bug.  I just want to ping @buck54321 about this to see if this rings a bell.

Reply:
Without looking too deeply into it, I would guess it's an artifact of an older implementation, or a feature that was never fully hashed out.

Reply:
Thanks.  I don't see any issues with this change and everything I've checked looks fine.